src/CoursePageResources/index.scss

```diff
@@ -1,48 +1,19 @@
-.container-fluid {
-    background-color: #EAF3FB;
-    color: #2E8ED3;
-}
-
-.container-fluid-container {
-    border-color: #B2D5EE;
-}
-
-.container-fluid-enabled-button {
-    border-color: #0075B4;
-    color: #2E8ED3;
-}
-
-.container-fluid-enabled-button:hover {
+.course-page-title {
+  border: 2px solid $white;
+  border-color: theme-color("info", 300);
+  border-radius: 5px; //Maybe this can be standardised?
+  .disabled {
+    border-color: theme-color("gray", 100);
+  }
+}
```

Comment:
I am wondering if this can all be standardised, and then this could just include the standard styling if it needed to be customised.